src/client/app/services/github.service.ts

```diff
@@ -95,7 +95,7 @@ export class GitHubService {
         if (this.profile == null) {
             return Observable.of([]);
         }
-        let url = `${this._baseUrl}/users/${this.profile.login}/gists`;
+        let url = `${this._baseUrl}/users/${this.profile.login}/gists?timestamp=${(new Date).getTime()}`;
```

Comment:
It looks like angular doesn't support cache busting by default like jQuery, but you could also add  'Cache-Control' : 'no-cache' to the headers to prevent the browser cache.